plonetheme/onegovbear/browser/dynamic_scss_resources.py

```diff
@@ -0,0 +1,23 @@
+from ftw.theming.interfaces import ISCSSResourceFactory
+from ftw.theming.resource import SCSSResource
+from plonetheme.onegovbear.browser.forms import ANNOTATION_KEY
+from zope.annotation import IAnnotations
+from zope.interface import provider
+
+
+@provider(ISCSSResourceFactory)
+def custom_scss_variables(context, request):
+
+    annotations = IAnnotations(context)
+    scss_variables = annotations.get(ANNOTATION_KEY)
+
+    source = ''
+    if scss_variables:
+        source = '; '.join(
+            ['{0}: {1}'.format(value['variable_name'], value['value'])
+             for value in scss_variables.itervalues()]
+        ) + ';'
+
+    return SCSSResource('plonetheme.onegovbear.custom.scss',
+                        slot='addon',
```

Comment:
This should go to the `variables` slot. The slot was created exactly for this purpose :wink: 
Is there any trouble with ordering?
If everything is correct it should not be relevant what the order is, since all other variable declarations should be declared with `!default`, which means we can insert this source (without `!default`s) before or after and it has precedence in both cases.